bosh/releases/pre_render_scripts/api/cloud_controller_ng/jobs/patch_cloud_controller_api_ctl.sh

```diff
@@ -1,27 +0,0 @@
-#!/usr/bin/env bash
-
-set -o errexit -o nounset
-
-target="/var/vcap/all-releases/jobs-src/capi/cloud_controller_ng/templates/cloud_controller_api_ctl.erb"
```

Comment:
Same here — this script appears to be unused.

Reply:
Squashed history and amended commit message based on these comments.
Commit 5f7cd28c48738106936b93d1ae215ead9c8d412d